app/actions/nodeNetworkActions.js

```diff
@@ -10,67 +11,60 @@ import {
 } from '../core/constants'
 
 const ID = 'nodeNetwork'
+const MAX_RESPONSE_TIME = 10000
+const RPC_PING_OVERRIDE = 10000
```

Comment:
can you explain these 2 variables. the last time I checked there was a timeout at 2 seconds in neon-js unless I am not remembering correctly. It has been a while

Reply:
10 seconds seems very excessive in any case

Reply:
Just reread the overview section. maybe use 5 seconds so the user gets a faster notice. I doubt there will be good responses after 5 seconds

Reply:
does the 3g work after 5 seconds?

Reply:
Sounds good I will test with 5 seconds... I agree that 10 could be a little excessive. Thanks for taking a look @dvd-schwrtz

Reply:
5 seconds works for slow 3g 👍